automatic/mp3tag/update.ps1

```diff
@@ -19,16 +19,25 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-    $download_page = Invoke-WebRequest -Uri $releases
+    $download_page = Invoke-WebRequest "$releases.json" | ConvertFrom-Json
+    $content = $download_page.post_stream.posts[0].cooked
 
-    $result = $download_page.Content -match "<h2>Mp3tag v([\d\.]+)([a-z])?</h2>"
+    $result = $content -match "<strong>\s*Version:\s*</strong>\s*([\d\.]+)([a-z])?"
 
-    if (!$Matches) {
+    if (!$result) {
       throw "mp3tag version not found on $releases"
     }
 
     $version = $Matches[1]
     $versionSuffix = $Matches[2]
+
+    $result = $content -match "<strong>\s*Status:\s*</strong>\s*<strong>\s*([a-z]+)\s*</strong>"
+
+    if (!$result) {
+      throw "mp3tag status not found on $releases"
```

Comment:
Maybe better to 'ignore' here instead err ? Would you really wont to get error record for this ?

Reply:
If the script won't know if status is Stable or Beta it may push the beta package without -beta suffix (so it would't  count as prerelease).
Possibly we could add -beta suffix for every status that isn't "Stable", but it still wouldn't be perfect. I can change it, I just want you to know that it's a possibility that update script will treat the version as not prerelease when it is or the other way (depending on what we choose to do).
From what I've seen, this format is kept for quite a while, so chances that it will not match in the first place are really small and if it don't match at some point, we would be able to fix it before pushing it as prerelease, if it's stable.

Reply:
It looks like you misunderstood. Returning 'ignore' will not update the package nor it will make it to throw error, it will show up in ignored section of the report along with your custom Write-Host message.

https://github.com/majkinetor/au#handling-update-errors

Reply:
Oh, sorry, I didn't know 'ignore' is a thing in au.